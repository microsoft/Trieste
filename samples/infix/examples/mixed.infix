x = 1 + 2 * 3 + 5.3 - 4 - 2 / 0.1;
y = 3.2 * x + 5;
print "x" x;
print "y" y;