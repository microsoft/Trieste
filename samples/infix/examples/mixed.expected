//! calculate
//! --enable-tuples calculate
//! --enable-tuples --tuples-require-parens calculate
//! --enable-tuples --tuples-require-parens --use-parser-tuples calculate
x <- -11.700000
y <- -32.440000
